scripts/zones/Oldton_Movalpolos/mobs/Goblin_Wolfman.lua

```diff
@@ -0,0 +1,39 @@
+-----------------------------------
+-- Area: Oldton Movalpolos (11)
+--  Mob: Goblin Wolfman
+-----------------------------------
+
+function onMobDeath(mob, player, isKiller)
+end;
+
+
+function onMobSpawn(mob)
+    local GoblinWolfmanID = mob:getID();
+
+   -- Set the hp% to use blood weapon
+    mob:setLocalVar("jobSpecTrigger", math.random(50, 75));
+
+end;
+
+function onMobFight(mob, target)
+    local trigger = mob:getLocalVar("jobSpecTrigger")
+    if (mob:getHPP() <= trigger and mob:getLocalVar("BloodWeapon") == 0) then
+        mob:useMobAbility(jobSpec.BLOOD_WEAPON);
+        mob:setLocalVar("BloodWeapon", 1);
+    end
+end;
+
+function onMobDespawn(mob)
+    local npc = GetNPCByID(16822469);
```

Comment:
You'll want to have the QM in MobIDs too.

Reply:
there we go, I learned something new today, thanks